app/services/converters/product_to_journal_rows.rb

```diff
@@ -0,0 +1,22 @@
+module Converters
+  class ProductToJournalRows
+
+    attr_accessor :journal, :product, :total
+
+    def initialize(journal, product, total)
```

Comment:
The original functionality required `total`.  With this object I think we have to require `total` to support the previous behavior.  Alternatively, with the order detail converter we can infer the `total` if an explicit value is not provided.